lib/adb.js

```diff
@@ -65,6 +68,8 @@ class ADB {
 ADB.createADB = async function createADB (opts) {
   let adb = new ADB(opts);
   await adb.getAdbWithCorrectAdbPath();
+  // get the api level to make sure that the device is reachable
+  await retryInterval(ADB_INITIAL_RETRIES, 100, adb.getApiLevel.bind(adb));
```

Comment:
should this behavior be controlled by `opts`?

Reply:
I don’t really see a reason to without more data on a problem. I’ve run it locally and through the uiautomator2 driver on travis. As I said, we always get the api level after getting an instance, so this is happening anyway in practical terms. And we know that retryInterval works as it should.

Reply:
I don't have a strong opinion on the current implementation. Just wanted to make sure it's more flexible.

Reply:
Short of adding a capability, any option is going to take work to get in, so I'd rather wait until there is actually a problem. It would, however, be easy enough to add if needed.